web/html/src/manager/admin/setup/products/products.js

```diff
@@ -1,972 +0,0 @@
-      pageContent = (
-        <div className='alert alert-warning' role='alert'>
-          {t('This server is configured as an Inter-Server Synchronisation (ISS) slave. Products can only be managed on the ISS master.')}
-        </div>
-      );
-    }
-
-    const prevStyle = { 'marginLeft': '10px' , 'verticalAlign': 'middle'};
-    const currentStepIndex = _SETUP_WIZARD_STEPS.indexOf(_SETUP_WIZARD_STEPS.find(step => step.active));
-    const footer =
-      <div className='panel-footer'>
-        <div className='btn-group'>  
-          {
-            currentStepIndex > 1 ?
-              <a className='btn btn-default' href={_SETUP_WIZARD_STEPS[currentStepIndex-1].url}>
-                <i className='fa fa-arrow-left'></i>{t('Prev')}
-              </a> : null
-          }
-          {
-            currentStepIndex < (_SETUP_WIZARD_STEPS.length - 1) ?
-              <a className='btn btn-success' href={_SETUP_WIZARD_STEPS[currentStepIndex+1].url}>
-                <i className='fa fa-arrow-right'></i>{t('Next')}
-              </a> : null
-          }
-        </div>
-        <span style={prevStyle}>
-          { currentStepIndex+1 }&nbsp;{t('of')}&nbsp;{ _SETUP_WIZARD_STEPS.length }
-        </span>
-      </div>;
-
-    return (
-      <div className='responsive-wizard'>
-        {title}
-        {tabs}
-        <div className='panel panel-default' id='products-content'>
-            <div className='panel-body'>
-              {pageContent}
-            </div>
-        </div>
-        {footer}
-      </div>
-    )
-  }
-}
-
-/**
- * Show the products data
-*/
-class Products extends React.Component {
-  state = {
-    popupItem: null,
-    archCriteria: [],
-    visibleSubList: []
-  };
-
-  componentDidMount() {
-    const currentObject = this;
-
-    //HACK: usage of JQuery here is needed to apply the select2js plugin
-    jQuery('select#product-arch-filter.apply-select2js-on-this').each(function(i) {
-      var select = jQuery(this);
-      // apply select2js only one time
-      if (!select.hasClass('select2js-applied')) {
-        select.addClass('select2js-applied');
-
-        var select2js = select.select2({placeholder: t('Filter by architecture')});
-        select2js.on("change", function(event) {
-          currentObject.handleFilterArchChange(select.val() || []);
-        });
-      }
-    });
-  }
-
-  getDistinctArchsFromData = (data) => {
-    var archs = [];
-    Object.keys(data).map((id) => data[id])
-        .forEach(function(x) { if (!archs.includes(x.arch)) archs.push(x.arch); });
-    return archs;
-  };
-
-  handleFilterArchChange = (archs) => {
-    this.setState({archCriteria: archs});
-  };
-
-  filterDataByArch = (data) => {
-    if(this.state.archCriteria.length > 0) {
-      return data.filter(p => this.state.archCriteria.includes(p.arch));
-    }
-    return data;
-  };
-
-  handleSelectedItems = (items) => {
-    this.props.handleSelectedItems(items);
-  };
-
-  handleUnselectedItems = (items) => {
```

Comment:
This file had a mess with a mixup of `handleUnselectedItems` and `handleUnSelectedItems` (different capitalization of `s/S`), where both were referenced in some places. I did best I could do test it and fix it up as seemed correct, but would be good if you could also give this a test please.